### DISCUSSION TITLE
Arrays & Structs

### DISCUSSION THREAD

**Discussion by ttshewang** [Cyfrin/foundry-full-course-cu#2291] [General]
I ran the same code as there in the course, somehow I am not able to get the same results as taught in the course...when contract is deployed it says that listOfPeople input required despite inserting the name and number...can anyone tell me what is wrong...compiler runs without an error 

// SPDX-License-Identifier: MIT
pragma solidity 0.8.18; // stating our verison 
 
 contract SimpleStorage {
    
    uint256  myFavoriteNumber; 
    
    struct Person {
        uint256 favoriteNumber;
        string name;
    }
  
    Person[] public listOfPeople;

    function store(uint256 _favoriteNumber) public {
        myFavoriteNumber = _favoriteNumber;
    } 

    function retrieve() public view returns(uint256) {
        return  myFavoriteNumber;

    }
        
        function addPerson(string memory _name, uint256 _favoriteNumber) public {
          listOfPeople.push(Person(_favoriteNumber, _name )); 
    } 
 }

**Comment by EngrPips** [comment]
Hello @ttshewang, Please run the deployment process again and screenshot the error you are getting so that we can know about the process and maybe help with the solution.
